src/components/home/Curve.tsx

```diff
@@ -0,0 +1,8 @@
+import curve from '../../images/rotten-curve-5.png';
+
+export const Curve = () => {
+   const CurveStyle = {
+      backgroundImage: `url(${curve})`,
+   };
+   return <div className="absolute w-full h-8 bottom-0 left-0 bg-cover bg-center bg-no-repeat" style={CurveStyle}></div>;
+};
```

Comment:
Reserve `PascalCase` for components only.

```suggestion
export const Curve = () => {
   const backgroundImage = `url(${curve})`;
   return <div className="absolute w-full h-8 bottom-0 left-0 bg-cover bg-center bg-no-repeat" style={{ backgroundImage }} />;
};
```